components/checkbox.tsx

```diff
@@ -1,9 +1,21 @@
 import { Root, Indicator } from '@radix-ui/react-checkbox'
+import type {
+  CheckboxProps,
+  CheckboxIndicatorProps,
+} from '@radix-ui/react-checkbox'
 
-export const Checkbox = (props: any) => (
-  <Root {...props} className={`${props.className}`}></Root>
+import styles from './checkbox.module.css'
+
+export const Checkbox = ({ className = '', ...props }: CheckboxProps) => (
+  <Root
+    {...props}
+    className={`relative w-24px h-24px border-1 border-cool-gray-300 rounded-sm after:(content absolute inset-0 -m-5px rounded-md border-2 border-red-600) ${styles.checkbox} ${className}`}
```

Comment:
Can we multi-line this?

Reply:
For some reason I'm unable to reply to the comment above about the focus outline. I am down to override the focus indicator styling [as long as it's cross-browser and accessible](https://coryrylan.com/blog/dont-override-css-outline-focus-styles). By all means feel free to make this part of a component ticket 😸

Reply:
@jackhajb feel free to multiline and organize classes on your own branches. eslint will respect your decisioins

i'm not going to do it on this branch, but i work in vue SFCs a lot where my eslint configs automatically add line breaks and group related classes, so i see where you're coming from

another thing to consider, that i haven't used much yet or made sure is working in our repos, is [attributify mode](https://windicss.org/posts/attributify.html). assuming its working, you could also switch over from `className` to attributes once you have enough styles to cause a line break, or need a more manageable way to bind

tl;dr theres wiggle room here

Reply:
@mis4mike I hear you on accessibility, let's hold off for now.  A custom focus outline would probably be best, but not really a priority at this point

Reply:
@replygirl Attributify mode seems interesting.  Would you rather have a line length of 200 char or 15 lines of attributes. Not sure my thoughts on that one.  The line break was a soft suggestion, so Mike will add or not as he sees fit.

Reply:
We probably do want to set a maximum line length. I don't have strong opinions yet - I think we can pick something that seems reasonable and see how it works with our code.

Reply:
i don't have a stake in it since we have 80ch globally enforced everywhere except long strings. going to mark as resolved and consider out of scope for this pr. if either of u want to extend the eslint config to give whitespace in jsx special treatment, feel free to do so in a different pr